lib/gossip/state_transitions.js

```diff
@@ -19,19 +19,20 @@
 // THE SOFTWARE.
 'use strict';
 
-function Suspicion(options) {
+var Member = require('../membership/member');
+
+function StateTransitions(options) {
     this.ringpop = options.ringpop;
-    this.period = options.suspicionTimeout ||
-        Suspicion.Defaults.suspicionTimeout;
-    this.logger = this.ringpop.loggerFactory.getLogger('suspicion');
+    this.periods = options.periods || {};
+    this.logger = this.ringpop.loggerFactory.getLogger('statetransitions');
 
     this.isStoppedAll = null;
     this.timers = {};
 }
 
-Suspicion.prototype.reenable = function reenable() {
+StateTransitions.prototype.reenable = function reenable() {
```

Comment:
There was a whole discussion in the ringpop-go repo about naming, one of the outcomes of that was to name this `enable`.